<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsLidarProfileGuiBase</class>
 <widget class="QMainWindow" name="QgsLidarProfileGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QgsMapCanvas" name="mMapCanvas"/>
  <widget class="QToolBar" name="mToolBarClassTools">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="mActionClassifyAboveLine"/>
   <addaction name="mActionClassifyBelowLine"/>
   <addaction name="mActionClassifyCloseToLine"/>
  </widget>
  <widget class="QToolBar" name="mToolBarControls">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="mActionPan"/>
   <addaction name="mActionZoomIn"/>
   <addaction name="mActionZoomOut"/>
   <addaction name="mActionZoomToLayer"/>
   <addaction name="mActionZoomLast"/>
   <addaction name="mActionZoomNext"/>
  </widget>
  <widget class="QStatusBar" name="mStatusBar"/>
  <widget class="QToolBar" name="mToolBarMoveProfile">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="mActionMoveProfileForward"/>
   <addaction name="mActionMoveProfileBackward"/>
  </widget>
  <action name="mActionClassifyAboveLine">
   <property name="text">
    <string>ClassifyAboveLine</string>
   </property>
   <property name="toolTip">
    <string>Classifies Lidar points above line</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="mActionClassifyBelowLine">
   <property name="text">
    <string>ClassifyBelowLine</string>
   </property>
   <property name="toolTip">
    <string>Classifies Lidar points below line</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="mActionClassifyCloseToLine">
   <property name="text">
    <string>ClassifyCloseToLine</string>
   </property>
   <property name="toolTip">
    <string>Classifies Lidar points close to line</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="mActionPan">
   <property name="text">
    <string>Pan</string>
   </property>
   <property name="toolTip">
    <string>Pan</string>
   </property>
  </action>
  <action name="mActionZoomIn">
   <property name="text">
    <string>ZoomIn</string>
   </property>
   <property name="toolTip">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="mActionZoomOut">
   <property name="text">
    <string>ZoomOut</string>
   </property>
   <property name="toolTip">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="mActionZoomToLayer">
   <property name="text">
    <string>ZoomToLayer</string>
   </property>
   <property name="toolTip">
    <string>Zoom to Layer</string>
   </property>
  </action>
  <action name="mActionZoomLast">
   <property name="text">
    <string>ZoomLast</string>
   </property>
   <property name="toolTip">
    <string>Zoom Last</string>
   </property>
  </action>
  <action name="mActionZoomNext">
   <property name="text">
    <string>ZoomNext</string>
   </property>
   <property name="toolTip">
    <string>Zoom Next</string>
   </property>
  </action>
  <action name="mActionMoveProfileForward">
   <property name="text">
    <string>MoveProfileForward</string>
   </property>
   <property name="toolTip">
    <string>Move profile forward for half of its depth </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
  <action name="mActionMoveProfileBackward">
   <property name="text">
    <string>MoveProfileBackward</string>
   </property>
   <property name="toolTip">
    <string>Move profile backward for half of its depth </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+5</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapCanvas</class>
   <extends>QWidget</extends>
   <header>qgsmapcanvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
